src/App.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
+import Layout from './Layout';
+import VersionInformation from './VersionInformation';
+
+import './Styles/styles.css';
```

Comment:
Writing all of the styles in one css file is wrong. Each compoennt should have its own.